src/index.html

```diff
@@ -5,8 +5,406 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header header_container">
+      <img
+        class="header__logo"
+        src="./images/logo.png"
+        alt="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a
+              class="nav__list_link is-active nav__list_link--active"
```

Comment:
окремі слова в назвах класів краще розділяти дефісом, бо в BEMі дефісом ще розділляють модифікатори
```suggestion
              class="nav__list-link is-active nav__list-link--active"
```